UI/ChatUI/src/.editorconfig

```diff
@@ -9,7 +9,7 @@ root = true
 ##########################################
 
 [*]
-indent_style = space
```

Comment:
@eriklimakc may be best to keep the current default indentation to make it consistent with the other samples

Reply:
Done ;D